<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCL_PROCESSOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="guiwidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_output">
     <property name="title">
      <string>另存为(output)</string>
     </property>
     <addaction name="actionSTL"/>
     <addaction name="actionOBJ"/>
     <addaction name="actionTXT"/>
     <addaction name="actionPLY"/>
     <addaction name="actionPCD"/>
    </widget>
    <addaction name="open_action"/>
    <addaction name="save_action"/>
    <addaction name="menu_output"/>
    <addaction name="separator"/>
    <addaction name="close_action"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>窗口</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>离群点移除</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>滤波</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>配准</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>关键点</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>表面重建</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>快捷键</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>5000</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>加载过的文件：</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhMultiLine</set>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>107</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>150</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>点云处理实时日志</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="log_textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>点云文件中包含的点数为：</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="1">
      <widget class="QPushButton" name="upward_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/up_view.png</normaloff>:/new/prefix1/resource/up_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="left_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/left_view.png</normaloff>:/new/prefix1/resource/left_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="main_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/main_view.png</normaloff>:/new/prefix1/resource/main_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="back_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/back_view.png</normaloff>:/new/prefix1/resource/back_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3" colspan="2">
      <widget class="QTextBrowser" name="textBrowser">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="top_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/top_view.png</normaloff>:/new/prefix1/resource/top_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="right_view">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/new/prefix1/resource/right_view.png</normaloff>:/new/prefix1/resource/right_view.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>X轴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y轴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z轴</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="comboBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>彩色渐变（默认）</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蓝色渐变</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红色渐变</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>黄色渐变</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="render_begin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>渲染</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="open_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>加载点云（open）</string>
   </property>
   <property name="toolTip">
    <string>打开点云文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="save_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存（save）</string>
   </property>
   <property name="toolTip">
    <string>保存点云文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="close_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭所有（close）</string>
   </property>
   <property name="toolTip">
    <string>关闭所有打开的点云文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="exit_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出程序（exit）</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPCD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PCD</string>
   </property>
  </action>
  <action name="actionPLY">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PLY</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionTXT">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TXT</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOBJ">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OBJ</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSTL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>STL</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
